src/version/version.go

```diff
@@ -26,6 +27,10 @@ var (
 
 // LogVersion logs metadata about the Operator.
 func LogVersion() {
+	LogVersionToLogger(log)
```

Comment:
Imho it would be better to leave the call of LogVersion in the main.go as it is called only once for all subcommands there and add an additional function call for the supportArchive command, with that we dont have to maintain the functionCall in 5 different places but in one.